src/components/structures/UserSettings.js

```diff
@@ -337,9 +337,6 @@ module.exports = React.createClass({
             },
         */
         ];
-
-        var syncedSettings = UserSettingsStore.getSyncedSettings();
```

Comment:
i'd expect that supporting location sharing would be precisely the sort of thing that the generic settings code would be good for.  (or possibly hiding it behind a labs option for now)